README.md

```diff
@@ -9,6 +9,7 @@ Exports spreadsheets to OneDrive
 The configuration `config.json` contains following properties in `parameters` key: 
 
 - `append` - bool (optional): if sheet exists, rows are appended to the end, default false
+- `bulkSize` - int (optional): number of the rows in one batch / insert API call, default `10 000`
```

Comment:
nemělo by to být spíš `batchSize`?

Reply:
moze byt, premenujem, ... nechal som povodny nazov, ktory sme tam v kode uz mali ... ale inde pouzivame `batchSize`.

Reply:
premenovane, mergujem ....